.github/workflows/qa.yaml

```diff
@@ -11,4 +11,4 @@ jobs:
         env:
           REQUIRE_DEV: true
         with:
-          args: analyse
+          args: analyse -c .phpstan/phpstan.neon.dist
```

Comment:
I've moved `phpstan` related things to `.phpstan` folder. Because It will be too much filles in the root (`phpstan.neon.dist`, `phpstan-baseline.neon`, 3 files that can't be resolved because of `CoreBundle`)